apisix/cli/ngx_tpl.lua

```diff
@@ -270,7 +270,6 @@ http {
 
     {% if use_apisix_openresty then %}
```

Comment:
Should move the whole block out from the access_log stuff...

Reply:
done